templates/KpisList/Kpi.jsx

```diff
@@ -0,0 +1,86 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { HvCard, HvTypography } from "@hitachivantara/uikit-react-core";
+import { TopXS, BottomXS } from "@hitachivantara/uikit-react-icons";
+
+/**
+ * A KPI.
+ *
+ * @param {instance} Object the instance returned by the `useHvData` data.
+ * @param {title} String The title of the KPI.
+ * @param {count} Integer The count of the KPI.
+ * @param {color} String The color used on the KPI header bar.
+ * @param {variation} String The value for the variation field.
+ * @param {status} Integer The status of the KPI.
+ */
+const Kpi = ({ title, count, color, variation, status, instance }) => {
+  return (
```

Comment:
Given the fact that this a template i would avoid using inline styling, to prevent the proliferation of inline styles when using it(this code is valid for the whole file to avoid spamming)